.github/workflows/continuous-integration.yaml

```diff
@@ -473,9 +484,6 @@ jobs:
         id: changed-files-in-pr
         if: | 
           env.IS_PR
-          && !fromJSON(env.IS_WEEKLY) 
```

Comment:
`IS_PR` is enough (all four states can't overlap) + it's simpler to read/understand the conditions